src/neptune/github.d

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+
+    Github interaction helpers
+
+    Copyright:
+        Copyright (c) 2017 Sociomantic Labs GmbH. All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
+module neptune.github;
+
+import neptune.api;
+import octod.api.repos;
+
+/// Structure defining a name-tag association
+struct Head
+{
+    string name;
+    string sha;
+}
+
+/*******************************************************************************
+
+    Params:
+        repo = repository object
+
+    Returns:
+        all branches for this repository
+
+*******************************************************************************/
+
+public Head[] getBranches ( ref Repository repo )
```

Comment:
In general I would like to fill `octod` repo with such functions on demand - i.e. after you find oneself in need of it, both put one in neptune for immediate usage and submit octod PR with copy of it.